src/qengine/opencl.cpp

```diff
@@ -1192,7 +1192,7 @@ real1 QEngineOCL::ProbMask(const bitCapInt& mask, const bitCapInt& permutation)
     std::vector<bitCapInt> skipPowersVec;
     for (length = 0; v; length++) {
         oldV = v;
-        v &= v - 1; // clear the least significant bit set
+        v &= v - ONE_BCI; // clear the least significant bit set
```

Comment:
Separate PR.

Reply:
Please, no, not now that you've reviewed it...